frontends/web/src/routes/lightning/components/payment-details.tsx

```diff
@@ -0,0 +1,141 @@
+import { TxDetailCopyableValues } from '@/components/transactions/components/address-or-txid';
+import { getTxSign } from '@/components/transactions/utils';
+import styles from '@/components/transactions/components/details.module.css';
+
+type TTxDetailsDialog = {
+  open: boolean;
+  onClose: () => void;
+  payment: IPayment;
+  sign: string;
+}
+
+export const PaymentDetailsDialog = ({
+  open,
+  onClose,
+  payment,
+  sign,
+}: TTxDetailsDialog) => {
+  const { t } = useTranslation();
+
+  return (
+    <Dialog
+      open={open}
+      title={t('transaction.details.title')}
+      onClose={onClose}
+      slim
+      medium>
+      {payment && (
+        <>
+          <TxDetail label="Memo">
+            {payment.description}
+          </TxDetail>
+          <TxDateDetail
+            time={new Date(payment.paymentTime * 1000).toString()}
+          />
+          <TxDetail label="Amount">
+            {payment.amountMsat * 0.001} sat
+          </TxDetail>
+          <TxDetail label={t('transaction.details.fiat')}>
+            <span className={styles.fiat}>
+              <FiatConversion
+                amount={{
+                  amount: `${payment.amountMsat * 0.001}`,
+                  unit: 'sat',
+                  estimated: false
+                }}
+                sign={sign}
+                noAction
+              />
+            </span>
+          </TxDetail>
+          <TxDetail label="fee">
+            {payment.feeMsat} Msat
+          </TxDetail>
+          <TxDetail label="type">
+            {payment.paymentType}
+          </TxDetail>
+          <TxDetail label="status">
+            {payment.status}
+          </TxDetail>
+          { payment.paymentType !== 'closedChannel' && (
+            <TxDetailCopyableValues
+              key="paymentPreimage"
+              label="paymentPreimage"
+              values={[
+                (payment.details.data as LnPaymentDetails).paymentPreimage
+              ]}
+            />
+          )}
+          { payment.paymentType !== 'closedChannel' && (
+            <TxDetailCopyableValues
+              key="paymentHash"
+              label="paymentHash"
+              values={[
+                (payment.details.data as LnPaymentDetails).paymentHash
+              ]}
+            />
+          )}
+        </>
+      )}
+    </Dialog>
+  );
+};
+
+type TTransactionDetails = {
+  id: string | null;
+  payment?: IPayment;
+  onClose: () => void;
+}
+
+export const PaymentDetails = ({
+  id,
+  payment,
+  onClose,
+}: TTransactionDetails) => {
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    if (id) {
+      setOpen(true);
+    }
+  }, [id]);
+
+  if (!payment) {
+    return null;
+  }
+
+  return (
+    <PaymentDetailsDialog
+      open={open}
+      onClose={() => {
+        setOpen(false);
+        onClose();
+      }}
+      payment={payment}
+      sign={getTxSign(payment.paymentType === 'sent' ? 'send' : 'receive')}
```

Comment:
Same here about closedChannel txs, which I think should be showed as "sent"

Reply:
I don't know how to test this, is it already possible to close a channel with the BBApp?